pkg/registries/docker/docker_integration_test.go

```diff
@@ -70,7 +70,7 @@ func TestOCIImageIndexManifestWithoutManifestCall(t *testing.T) {
 				Endpoint: "https://gcr.io",
 			},
 		},
-	}, false)
+	}, false, nil)
```

Comment:
do we have a test where it's non nil and verify it's correctly used?

Reply:
we do now